torchmultimodal/modules/layers/attention.py

```diff
@@ -0,0 +1,143 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+import itertools
+from typing import Optional
+
+import numpy as np
+
+import torch
+import torch.nn as nn
+
+from torchmultimodal.utils.common import tensor_slice
+
+
+class BroadcastPositionEmbedding(nn.Module):
```

Comment:
nit: maybe `BroadcastedPositionEmbedding`? broadcast is a verb and it feels weird to use in class name